src/i18n/en/openSystem.json

```diff
@@ -1578,7 +1578,7 @@
       },
       "createMarket": {
         "name": "Create Market",
-        "description": "<h3>Rune Spell</h3><p><i><b>Cost:</b> 3+<br><b>Type:</b> Ritual<br><b>Cults:</b> Issaries</i></p><p>Requires four wands. Creates a neutral trading ground 10 meters square, warded against vs. those with hostile intent. This provides the benefits of @JournalEntry[${{SS.module.name}}$.spirit-magic-descriptions-DUMMY]{Countermagic} 1,  @JournalEntry[${{SS.module.name}}$.spirit-magic-descriptions-DUMMY]{Spirit Screen} 2, and does 1D3 points of damage (ignoring armor) to a random hit location of an unwelcome intruder. Additional Rune points spent can increase the magical effects by the same amounts per point, or they can increase the size of the market by a multiple equal to those points spent. Lasts eight weeks.</p>"
+        "description": "<h3>Rune Spell</h3><p><i><b>Cost:</b> 3+<br><b>Type:</b> Ritual<br><b>Cults:</b> Issaries</i></p><p>Requires four wands. Creates a neutral trading ground 10 meters square, warded against vs. those with hostile intent. This provides the benefits of @JournalEntry[${{openSystem.module.name}}$.spirit-magic-descriptions-DUMMY]{Countermagic} 1,  @JournalEntry[${{openSystem.module.name}}$.spirit-magic-descriptions-DUMMY]{Spirit Screen} 2, and does 1D3 points of damage (ignoring armor) to a random hit location of an unwelcome intruder. Additional Rune points spent can increase the magical effects by the same amounts per point, or they can increase the size of the market by a multiple equal to those points spent. Lasts eight weeks.</p>"
```

Comment:
This kind of translated links does not work as it is now - and I think it will be hard to make the openSystem.json file translate itself so to speak. Let's leave this as it is, and I can think about how to manage this.

Reply:
That has been there since before I started on the project. We do need to work how how add hyperlinks via RQIDs. We don't need to translate them, just link. Is that supported yet?

Reply:
Not yet with RQID links - I plan to implement that in Foundry10 since they've changed stuff regarding that there.